test/fail_compilation/fail19181.d

```diff
@@ -0,0 +1,14 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail19181.d(13): Error: undefined identifier LanguageError
+---
+*/
+struct S {
```

Comment:
Please follow the standard D style, curly braces on their own lines.

Reply:
fixed